bindings/pyfiction/include/pyfiction/algorithms/verification/design_rule_violations.hpp

```diff
@@ -50,16 +52,26 @@ inline void design_rule_violations(pybind11::module& m)
 
     py::class_<fiction::gate_level_drv_params>(m, "gate_level_drv_params")
         .def(py::init<>())
-        .def_readwrite("unplaced_nodes", &fiction::gate_level_drv_params::unplaced_nodes)
-        .def_readwrite("placed_dead_nodes", &fiction::gate_level_drv_params::placed_dead_nodes)
-        .def_readwrite("non_adjacent_connections", &fiction::gate_level_drv_params::non_adjacent_connections)
-        .def_readwrite("missing_connections", &fiction::gate_level_drv_params::missing_connections)
-        .def_readwrite("crossing_gates", &fiction::gate_level_drv_params::crossing_gates)
-        .def_readwrite("clocked_data_flow", &fiction::gate_level_drv_params::clocked_data_flow)
-        .def_readwrite("has_io", &fiction::gate_level_drv_params::has_io)
-        .def_readwrite("empty_io", &fiction::gate_level_drv_params::empty_io)
-        .def_readwrite("io_pins", &fiction::gate_level_drv_params::io_pins)
-        .def_readwrite("border_io", &fiction::gate_level_drv_params::border_io)
+        .def_readwrite("unplaced_nodes", &fiction::gate_level_drv_params::unplaced_nodes,
+                       DOC(fiction_detail_gate_level_drvs_impl_unplaced_nodes_check))
```

Comment:
For all these attributes, the docstring is referring to the functions inside the `detail` namespace, but not the `gate_level_drv_params` attributes, or?